plugins/downloader/youtube-dl.js

```diff
@@ -24,7 +24,14 @@ const ffmpeg = createFFmpeg({
 });
 const ffmpegMutex = new Mutex();
 
-const downloadVideoToMP3 = (
+function noTopic(channelName) {
```

Comment:
Great idea about `cleanupArtistName` but is that all you meant by sanitizing metadata in provider?
or were you also talking about other stuff?

Reply:
Looks good to me, but I think that having the variable available in advanced settings could be useful (instead of hardcoding it).
btw. I wouldn't mix none semicolon with one semicolon in the same file, but I'm sure this wasn't intended.

Reply:
I don't really see the point in allowing users to change that list, since it will almost never have any effect anyways..

But what did you mean by mixing semicolons? just what was fixed in 1140c3e or something else?